public/app/plugins/datasource/grafana/components/AnnotationQueryEditor.tsx

```diff
@@ -0,0 +1,133 @@
+import React from 'react';
+import { SelectableValue } from '@grafana/data';
+import { FieldSet, InlineField, InlineFieldRow, InlineSwitch, Select } from '@grafana/ui';
+import { css } from '@emotion/css';
+
+import { TagFilter } from 'app/core/components/TagFilter/TagFilter';
+import { GrafanaAnnotationQuery, GrafanaAnnotationType, GrafanaQuery } from '../types';
+import { getAnnotationTags } from 'app/features/annotations/api';
+
+const filterTooltipContent = (
+  <ul>
+    <li>
+      Dashboard: This will fetch annotation and alert state changes for whole dashboard and show them only on the event
+      {"'"}s originating panel.
+    </li>
+    <li>Tags: This will fetch any annotation events that match the tags filter.</li>
+  </ul>
+);
+
+const matchTooltipContent =
+  'By default Grafana only shows annotations that match all tags in the query. Enabling this returns annotations that match any of the tags in the query.';
+
+const tagsTooltipContent = (
+  <div>
+    A tag entered here as {"'"}foo{"'"} will match
+    <ul>
+      <li>
+        annotation tags {"'"}foo{"'"}
+      </li>
+      <li>
+        annotation key-value tags formatted as {"'"}foo:bar{"'"}
+      </li>
+    </ul>
+  </div>
```

Comment:
This produces weirdly formatted list in the UI:

![image](https://user-images.githubusercontent.com/2376619/124440890-68914980-dd7b-11eb-9664-25ae26892be2.png)

Reply:
ah yes, i noticed that and forgot to do anything about it, thanks! 😂 
we override the padding on all `ul`/`ol` here: https://github.com/grafana/grafana/blob/main/public/sass/base/_type.scss#L50

we can remove that, but i'm not sure what else it affects 😕  it must have been added for a reason